api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -187,7 +189,7 @@ message AccessLogCommon {
   // If upstream connection failed due to transport socket (e.g. TLS handshake), provides the
   // failure reason from the transport socket. The format of this field depends on the configured
   // upstream transport socket. Common TLS failures are in
-  // :ref:`TLS trouble shooting <arch_overview_ssl_trouble_shooting>`.
+  // :ref:`TLS Troubleshooting <arch_overview_ssl_trouble_shooting>`.
```

Comment:
nit: titles should only capitalize the first word

Reply:
Fixed :)